### ISSUE TITLE
This script is an exact copy of thelinuxchoice/saycheese

### ISSUE THREAD

**Issue by BullsEye0** [SharqanAhamed/shasnap#2]
How difficult is it to mention the owner of this script.. You did not make this .. 
https://github.com/thelinuxchoice/saycheese

**Comment by Waxxx333** [comment]
What a scumbag move. End your pathetic existence. Skids like you have no place in this world.

**Comment by SharqanAhamed** [comment]
Yeah yes I accept that but I mentioned my html script... And then Linux always open source right??
